apps/src/netsim/NetSimRouterNode.js

```diff
@@ -606,11 +607,18 @@ NetSimRouterNode.prototype.onMessageTableChange_ = function (rows) {
   }
 };
 
+var packetEncoder = new PacketEncoder([
+  { key: 'toAddress', bits: 4 },
+  { key: 'fromAddress', bits: 4 },
+  { key: 'payload', bits: Infinity }
+]);
```

Comment:
I know this needs to live in a common place (maybe the level config?) but I haven't figured out where yet.